var/spack/repos/builtin/packages/python/package.py

```diff
@@ -21,14 +20,16 @@
 from llnl.util.lang import dedupe
 
 from spack.build_environment import dso_suffix, stat_suffix
+from spack.build_systems.autotools import AutotoolsBuilder
+from spack.build_systems.generic import GenericBuilder
```

Comment:
I'd avoid this, for clarity. Can we do:
```python
from spack.build_system import autotools, generic
```
and below use e.g. `autotools.AutoolsBuilder` for the base class? That would make the name unambiguous and reduce the amount of typing wrt fully qualified namespaces.

Reply:
I don't really see it as ambiguous and this is the syntax I've used in the packages I've written. I've never needed to access a property of the super class, and if I did, I'd probably use super().

Reply:
You rebind a name that was imported from another module. I find that ambiguous,  since the same name is set two times in the same global scope. The change proposed above reduce the typing to a minimum and removes the ambiguity, so I'd go for that.